### PR TITLE
feat(server): added layers controller skeleton

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

- updated mc-logger
- updated mc-models
- fixed relative openapi support
- added layers controller

### CODE REVIEW

File: src/common/interfaces.ts [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import { ILogMethod } from '@map-colonies/mc-logger';
-
 export interface ILogger {
-  log: ILogMethod;
+  log: (level: string, message: string) => void;
```
Comment (asafMasa) [inline src/common/interfaces.ts:2]: why?

File: tests/integration/layers/layers.spec.ts [hunk 2]
```diff
@@ -15,16 +14,14 @@ describe('resourceName', function () {
...
-      const response = await requestSender.getResource();
-
-      expect(response.status).toBe(httpStatusCodes.OK);
-
-      const resource = response.body as IResourceNameModel;
-      expect(resource.id).toEqual(1);
-      expect(resource.name).toEqual('ronin');
-      expect(resource.description).toEqual('can you do a logistics run?');
-    });
+    // it('should return 200 status code and the resource', async function () {
```
Comment (asafMasa) [inline tests/integration/layers/layers.spec.ts:17]: why it is disabled? or commented

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: tests/integration/layers/layers.spec.ts (hunk 2 above)
Comment (cl-shaharshavit-rnd) [reply tests/integration/layers/layers.spec.ts:17]: I kept is a template for feature test.
there is no logic yet so noting to test

File: src/common/interfaces.ts (hunk 1 above)
Comment (cl-shaharshavit-rnd) [reply src/common/interfaces.ts:2]: 1) mc-logger 2.0.0 no longer exposes this interface so it cant be used
2) this changes decouples the service from the logger properly, and this decoupling is probably the reason for this interface in the first place

**Review by asafMasa — APPROVED** [review]

**Pull request merged**